void main () {
  try {
  int number = 10 ~/ 3;
  print(number);
  } catch (e) {
  print('Error: $e');
}
}